dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionMetadata.java

```diff
@@ -41,7 +41,7 @@
     protected boolean verified;
 
     @Column()
-    protected String verifiedSource;
+    protected String verifiedSource = "[]";
```

Comment:
This is an unusual default, don't see if there's a migration yet but it seems like without one existing data will be null/empty.